.github/dependabot.yml

```diff
@@ -168,6 +168,9 @@ updates:
     schedule:
       interval: "daily"
     open-pull-requests-limit: 20
+    ignore:
+      # The instrumentation only supports specific versions of GraphQL and the test app needs to test against a specific version
+      - dependency-name: "GraphQL*"
```

Comment:
Should it not be for `/test/test-applications/integrations/TestApplication.GraphQL` instead of `/test/test-applications/integrations/dependency-libs/TestApplication.ExampleLibrary`?

Reply:
You can change default branch to `IgnoreGraphQLUpdates` on your fork and see what dependabot will do.